sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,227 @@
+# July'21 release notes
+
+Release notes for commit range 6a49170027fb..962909fe9e78
+
+## New features
+ - Implement SYCL2020 specialized constants [07b27965] [ba3d657] [bd8dcf4]
+   [d15b841]
+ - Provide SYCL 2020 function objects [24a2ad89]
+ - Added support for ITT notification in SYCL Runtime [a7b8daf] [8d3921e3]
+ - Implement SYCL 2020 sub_group algorithms [e8caf6c3]
+ - Implement SYCL-2020 `sycl::handler::host_task` method [75e5a269]
```

Comment:
NIT: There's inconsistent spelling of "SYCL 2020"/"SYCL-2020"/"SYCL2020". I would suggest using "SYCL 2020" everywhere.